src/foam/u2/Element.js

```diff
@@ -2648,6 +2648,17 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'digColumns',
```

Comment:
digColumns has nothing to do with u2. Undo these changes. We'll introduce digColumns to models when we have everything else working.